tests/reconfiguration.py

```diff
@@ -78,10 +84,19 @@ def test_add_node_from_backup(network, args):
 
 @reqs.description("Adding a valid node from snapshot")
 @reqs.at_least_n_nodes(2)
-@reqs.add_from_snapshot()
 def test_add_node_from_snapshot(
     network, args, copy_ledger_read_only=True, from_backup=False
 ):
+    # Before adding the node from a snapshot, override at least one app entry
```

Comment:
Moving this from `test_requirements.py` which was only used historically to avoid code duplication.